pkgs/os-specific/darwin/apple-sdk/frameworks.nix

```diff
@@ -107,7 +107,7 @@ with frameworks; with libs; {
   Tcl                     = [];
   VideoDecodeAcceleration = [ CoreVideo ];
   VideoToolbox            = [ CoreMedia CoreVideo ];
```

Comment:
Notice typo in the commit message `s/applw/apple/`

Reply:
Fixed